spec/requests/status_spec.rb

```diff
@@ -7,17 +7,7 @@
     it 'returns a status message' do
       get('/status')
       expect(response).to be_successful
-<<<<<<< HEAD
```

Comment:
Ideally the marker changes should be squished into commit d9516f1429ad0ec84b5e64a903d485b8b6038d97.

The README change feels separate to the marker fixes, might make sense being it's own commit with a relevant message.

"Manually fixed rebase" doesn't feel like a good commit message for somebody trying to understand the documentation change.